src/index.html

```diff
@@ -17,11 +17,399 @@
+          <div class="review">
+            <div class="stars stars--4">
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+            </div>
+            <p class="review__reviews">Reviews: 5</p>
+            <span class="review__price">Price:</span>
+            <span class="review__numbers">$2,199</span>
+          </div>
+          <a
+            href="#"
+            class="card__button"
+            data-qa="card-hover"
+          >
+            BUY
+          </a>
+        </div>
+      </div>
+
+      <!-- 7-card -->
+      <div
+        class="card"
+        data-qa="card"
+      >
+        <img
+          class="card__image"
+          src="./images/imac.jpeg"
+          alt="img-computer"
+        />
+        <div class="card__bottom">
+          <h2 class="card__title">
+            APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+          </h2>
+          <p class="card__product-code">Product code: 195434</p>
+          <div class="review">
+            <div class="stars stars--4">
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+            </div>
+            <p class="review__reviews">Reviews: 5</p>
+            <span class="review__price">Price:</span>
+            <span class="review__numbers">$2,199</span>
+          </div>
+          <a
+            href="#"
+            class="card__button"
+            data-qa="card-hover"
+          >
+            BUY
+          </a>
+        </div>
+      </div>
+
+      <!-- 8-card -->
+      <div
+        class="card"
+        data-qa="card"
+      >
+        <img
+          class="card__image"
+          src="./images/imac.jpeg"
+          alt="img-computer"
+        />
+        <div class="card__bottom">
+          <h2 class="card__title">
+            APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+          </h2>
+          <p class="card__product-code">Product code: 195434</p>
+          <div class="review">
+            <div class="stars stars--4">
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+            </div>
+            <p class="review__reviews">Reviews: 5</p>
+            <span class="review__price">Price:</span>
+            <span class="review__numbers">$2,199</span>
+          </div>
+          <a
+            href="#"
+            class="card__button"
+            data-qa="card-hover"
+          >
+            BUY
+          </a>
+        </div>
+      </div>
```

Comment:
The repeated `card` blocks suggest that you should be using a loop to generate these in your HTML template instead of writing them out manually. If this is a static HTML file, disregard this comment.